File: xyident/baekjoon/9단계/5086.js
let input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

for(let i = 0; i < input.length-1; i++){
    const num = input[i].split(' ');

    if(num[0] % num[1] == 0){
        console.log("multiple");
    } else if (num[1] % num[0] == 0){
        console.log("factor");
    } else {
        console.log("neither");
    }
}

File: rolemadelen/baekjoon/step9/5086.rs
use std::io::{stdin};

fn read() -> (i32, i32) {
    let mut input = String::new();
    stdin().read_line(&mut input).unwrap();
    let mut input = input.trim().split_whitespace();

    let a: i32 = input.next().unwrap().parse().unwrap();
    let b: i32 = input.next().unwrap().parse().unwrap();

    (a, b)
}

fn main() {
    loop {
        let (a, b) = read();

        if a == 0 && b == 0 {
            break;
        }

        if b % a == 0 {
            println!("factor");
        } else if a % b == 0 {
            println!("multiple");
        } else {
            println!("neither");
        }

    }
}
